### PR TITLE
ui: add component to delete recordings from session

### PR DESCRIPTION
**Pull request by noreng-jg**

### CODE REVIEW

**Review by leonardojoao — DISMISSED** [review]

File: ui/src/components/session/SessionDeleteRecord.vue [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            mdi-playlist-remove
+          </v-icon>
+        </span>
+      </template>
+
+      <div>
+        <span v-if="isOwner">
+          Delete session record
+        </span>
+        <span
```
Comment (leonardojoao) [inline ui/src/components/session/SessionDeleteRecord.vue:20]: <span v-else>

Comment (leonardojoao) [inline ui/src/components/session/SessionDeleteRecord.vue:20]: <span v-else>

Comment (leonardojoao) [reply ui/src/components/session/SessionDeleteRecord.vue:20]: <span v-else>

Comment (leonardojoao) [inline ui/src/components/session/SessionDeleteRecord.vue:20]: Can be on the same line

**Review by leonardojoao — APPROVED** [review]

**Review by otavio — APPROVED** [review]

**Pull request merged**